CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.4.2
+
+_Feb 14, 2022_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- ❌ @mbrookes removed the Framer components (#30983)
```

Comment:
Should we give a bit of an explanation. For example, something like, we don't have the bandwidth to support it, so we are leaving it in the hands of the community. Maybe @mbrookes can provide better information.

Reply:
Right, it makes sense. I'll wait for Matt's response and do the release later today.

Reply:
I have updated the description of the PR to includes more details #30983. I wouldn't have an highlight on it to be honest, developers wouldn't care.

Reply:
I will remove it from the highlights then and do the release.

Reply:
```suggestion
```